### ISSUE TITLE
Alt text editor 

### ISSUE THREAD

**Issue by LukaszH77** [pixelfed/pixelfed-rn#87]
The current (.65) alt text editor wastes a lot of space and does not allow for convenient description of photos.

<img src="https://github.com/user-attachments/assets/c9c3454b-6b11-498e-98bd-caa4a1a8a456" width="250" />

Maybe you could move the text field lower and enlarge it, and permanently place a thumbnail of the photo at the top so that you could see what you were describing?

<img src="https://github.com/user-attachments/assets/effb62c5-ec42-46a4-9f03-db016e5a89fe" width="250" />

**Comment by Skivling** [comment]
This would benefit from a slightly smaller font size as well.

**Comment by Kwasow** [comment]
The image that the alt text is being added to should be visible now, but I agree we can make the font smaller, so I'll keep the issue open
